nakamoto/roster.json

```diff
@@ -45,7 +45,16 @@
     },
     {
       "name": "Beth Field",
-      "github": "BethField"
+      "github": "BethField",
+      "materials": [
+        {
+          "type": "cv"
+        },
+        {
+         "type": "video",
+          "url": "https://www.youtube.com/watch?v=ac-jJPTaSeM"
```

Comment:
Can you please change "url": "https://www.youtube.com/watch?v=ac-jJPTaSeM" to "url": "ac-jJPTaSeM"

Reply:
Oh sorry my bad I've changed it now